### PR TITLE
docs(enterprise): update statusbar plugin doc including preferences

### PR DESCRIPTION
**Pull request by jcesarmobile**

### CODE REVIEW

File: src/pages/enterprise/statusbar.md [hunk 1]
```diff
@@ -1,13 +1,33 @@
...
-version: 2.4.2
-minor: 2.4.X
+version: 2.5.0
+minor: 2.5.X
 ---
 
 The Status Bar plugin provides the ability to customize the appearance of the native mobile status bar, including setting visibility, background color, and more.
 
-<native-ent-install plugin-id="statusbar" variables=""></native-ent-install>
+<native-ent-install plugin-id="statusbar" variables="" capacitor="false"></native-ent-install>
```
Comment (dotNetkow) [inline src/pages/enterprise/statusbar.md]: remove the capacitor attribute.  Decided to use `capacitor-slug="filesystem"` if incompatible. If it is, simply don't apply the attribute

Comment (jcesarmobile) [reply src/pages/enterprise/statusbar.md]: I know, this is previous to your changes

Comment (jcesarmobile) [reply src/pages/enterprise/statusbar.md]: BTW, I've updated it

**Comment by jcesarmobile** [comment]
closing in favor of https://github.com/ionic-team/ionic-docs/pull/1123

**Pull request closed without merging**